examples/StripeAchCheckout.html

```diff
@@ -148,6 +148,14 @@
         </label>
     </div>
 
+
+    <div class="form-row">
+        <label>
+            <span>Make Default</span>
+            <input type="checkbox" bf-data="defaultPaymentMethod" />
```

Comment:
we use a different casing convention for fields provided in forms:

![image](https://cloud.githubusercontent.com/assets/9741519/14527594/fd0a5440-0242-11e6-8116-3e0ec0e79bad.png)

The user should input `use-as-default-payment-method`, and your implementation should map this to `defaultPaymentMethod` before sending it to the API.